Resources/Prototypes/_SpaceCats/Entities/Objects/Misc/pens.yml

```
@@ -0,0 +1,370 @@
+    sprite: _SpaceCats/Objects/pens.rsi
+    state: engineering2
+
+- type: entity
+  name: cargo fountain pen
+  parent: Pen
+  id: PenCargo1QM
+  description: The quartermaster's fountain pen, it seems to be gilded. Your employees shouldn't know where you got it from.
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Objects/pens.rsi
+    state: cargo2
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)